core/embed/extmod/modtrezorcrypto/modtrezorcrypto-secp256k1_zkp.h

```diff
@@ -364,11 +630,27 @@ STATIC const mp_obj_type_t mod_trezorcrypto_secp256k1_context_type = {
     .locals_dict = (void *)&mod_trezorcrypto_secp256k1_context_locals_dict,
 };
 
+/// def allocate_scratch_buffer() -> bytearray
+///     '''
+///     Allocate a buffer, large enough for holding a range/surjection proof.
+///     '''
+STATIC mp_obj_t mod_trezorcrypto_secp256k1_zkp_allocate_scratch_buffer() {
```

Comment:
(same here for "scratch")

Reply:
Fixed at https://github.com/romanz/trezor-firmware/pull/1/commits/f0ef2724ae5100cb7c797fb2ffde425d40be563f